ncm-systemd/src/main/resources/regular/tests/profiles/unitfile.pan

```diff
@@ -52,6 +52,7 @@ bind "/unitfile" = systemd_unitfile_config[];
         'TTYVHangup', false,
         'LimitSTACK', -1,
         'LimitNPROC', 100,
+        'ExecStartPre', list('/usr/bin/true', '/bin/foo'),
```

Comment:
It would be nice to include a command with a special character here, e.g. `-/bin/false`

```suggestion
        'ExecStartPre', list('/usr/bin/true', '-/bin/false'),
```